### DISCUSSION TITLE
Question: Does this schematics integrate Tailwind CSS + Nrwl Nx Workspace + Angular?

### DISCUSSION THREAD

**Discussion by svarunbe** [notiz-dev/ngx-tailwind#7] [Q&A] [answered]

**Comment by marcjulian** [comment]
@svarunbe Hi I am not sure if this schematics works with Nrwl Nx Workspace. I haven't tried as I am not using Nrwl Nx Workspace. Please try and let me know, if it doesn't support nx workspace you could provide a PR.

**Comment by SSmale** [comment, marked as answer]
this worked out the box for a latest nx project with angular 11
